metron-analytics/metron-profiler-common/src/main/java/org/apache/metron/profiler/DefaultMessageDistributor.java

```diff
@@ -146,16 +146,14 @@ public DefaultMessageDistributor(
   /**
    * Distribute a message along a MessageRoute.
    *
-   * @param message The message that needs distributed.
-   * @param timestamp The timestamp of the message.
    * @param route The message route.
    * @param context The Stellar execution context.
    */
   @Override
-  public void distribute(JSONObject message, long timestamp, MessageRoute route, Context context) {
+  public void distribute(MessageRoute route, Context context) {
```

Comment:
The `MessageRoute` abstraction now contains both the message (a JSONObject) and the timestamp (a Long).  Previously this was passed along separately.  It is much simpler for all of the ports (Spark included) to wrap this all into the `MessageRoute`.